src/utilities/cmdUtils.ts

```diff
@@ -0,0 +1,94 @@
+import { ChatInputCommandInteraction, REST, Routes } from "discord.js";
+import { Balance } from "../commands/balance";
+import { Cat } from "../commands/cat";
+import { DM } from "../commands/dm";
+import { Mock } from "../commands/mock";
+import { Ping } from "../commands/ping";
+import { Poll } from "../commands/poll";
+import { Roll } from "../commands/roll";
+
+//individual command cooldown/rate limit
+const cooldown = new Map<string, Map<string, number>>();
+const cooldownTime = 3000;
+
+//List of all commands
+const CMD_LIST = [
+    Ping,
+    Mock,
+    Cat,
+    Poll,
+    Balance,
+    Roll,
+    DM
+];
+/**
+ * Function to initialize all of the bots commands (making them usable/visable to discord users)
+ * Also uses to add choice options for commands that need it.
+ * 
+ * @param {string} token - bot token
+ * @param {string} clientID - bots discord account ID
+ * @param {string} guildID - discord server ID
+ */
+export async function initCmds(token: string, clientID: string, guildID: string): Promise<void> {
+    const rest = new REST({ version: '10' }).setToken(token);
+
+    Poll.addChoiceOptions();
+    DM.addChoiceOptions();
+
+    const cmdInfo = CMD_LIST.map(cmd => cmd.info.toJSON());
+    await rest.put(
```

Comment:
Think about making a similar function (or refactoring this one) to allow for deploying commands globally